common/national_focus/JER.txt

```diff
@@ -1807,15 +1815,43 @@ focus_tree = {
 
 		completion_reward = {
 			log = "[GetDateText]: [Root.GetName]: Focus JER_final_showdown"
-			hidden_effect = {
-				create_wargoal = {
+			if = {
+				limit = {
+					GRD = { exists = yes }
+				}
+				hidden_effect = {
+					create_wargoal = {
+						type = annex_everything
+						target = GRD
+					}
+				}
+				declare_war_on = {
 					type = annex_everything
 					target = GRD
 				}
 			}
-			declare_war_on = {
-				type = annex_everything
-				target = GRD
+			if = {
+				limit = {
+					NOT = { 382 = { owns_or_subject_of = yes } } #JER does not own Griffenheim (directly or indirectly through a subject)
+					NOT = { 382 = { owner = { has_war_with = ROOT } } } #JER is not already at war with the owner of Griffenheim (in some weird case where GRD exists but the owner of 382 is not GRD, we want to declare on BOTH, which is why we're using this 'if' condition instead of an 'else' block a few lines earlier)
+					NOT = { GRD = { exists = yes owns_state = 382 } } #if GRD exists and owns Griffenheim, then we're already declaring war on them in the above block
+				}
+				382 = { #JER declares war on the owner of Griffenheim
+					owner = {
+						ROOT = {
+							hidden_effect = {
+								create_wargoal = {
+									type = annex_everything
+									target = PREV
+								}
+							}
+							declare_war_on = {
+								type = annex_everything
+								target = PREV
+							}
+						}
+					}
+				}
```

Comment:
```suggestion

				hidden_effect = {
					create_wargoal = {
						type = annex_everything
						target = 382.owner
					}
				}
				declare_war_on = {
					type = annex_everything
					target = 382.owner
				}
```
can't remember off the top of my head if you can use variables in those two effects but I think that may give a nicer tooltip  (and less nesting)